cmd/restic/cmd_forget.go

```diff
@@ -91,6 +92,14 @@ func runForget(opts ForgetOptions, gopts GlobalOptions, args []string) error {
 	}
 	snapshotGroups := make(map[string]restic.Snapshots)
 
+	var GroupByTag      bool
+	var GroupByHost     bool
+	var GroupByPath     bool
+
+	GroupByTag = strings.Contains( opts.GroupBy, "tag" )
```

Comment:
That is very vague. I'd go for splitting `opts.GroupBy` by `,`, then iterate over the results, and use a `switch` statement that reacts on the strings `tag`, `host` and `path` and returns an error if the string is unknown.

What do you think?